### PR TITLE
feat(url-param): add optional url param for overriding Mapzen Search …

### PR DESCRIPTION
**Pull request by landonreed**
…with custom Pelias instance

#39

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const searchUrl = `${mapzenUrl}/search`
  *   {@link https://mapzen.com/documentation/search/autocomplete/#layers|layer types}
  * @param  {string} [$0.sources='gn,oa,osm,wof']
  * @param  {string} $0.text                       query text
+ * @param {string} $0.url                         optional URL to override Mapzen autocomplete endpoint
```
Comment (evansiroky) [inline index.js]: Please format the jsdoc with the optional parameter style.  The `sources` parameter is formatted in that way.

**Comment by evansiroky** [comment]
Please also run `npm run generate-docs` to update the README.

**Review by evansiroky — CHANGES_REQUESTED** [review]
Looks good, just update the docs as noted.

**Comment by landonreed** [comment]
Thanks @evansiroky.  Changes are in.

File: index.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ const searchUrl = `${mapzenUrl}/search`
  *   {@link https://mapzen.com/documentation/search/autocomplete/#layers|layer types}
  * @param  {string} [$0.sources='gn,oa,osm,wof']
  * @param  {string} $0.text                       query text
+ * @param {string} [$0.url]                       optional URL to override Mapzen autocomplete endpoint
```
Comment (evansiroky) [inline index.js]: Since there is a default value for this parameter, please make a notation of it.

**Review by evansiroky — CHANGES_REQUESTED** [review]
Please update the README again.

**Review by evansiroky — APPROVED** [review]
👍

**Pull request merged**